hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: hypershift-dvo-selectorsyncset-template
+parameters:
+  - name: REGISTRY_IMG
+    required: true
+  - name: IMAGE_TAG
+    required: true
+  - name: IMAGE_DIGEST
+    required: true
+  - name: REPO_NAME
+    value: deployment-validation-operator
+    required: true
+  - name: DISPLAY_NAME
+    value: Deployment Validation Operator
+    required: true
+  - name: CHANNEL
+    value: "alpha"
+    displayName: OLM subscription channel
+    description: OLM subscription channel
+    required: true
+  - name: TARGET_ORG
+    displayName: Target organization
+    description: Target api.openshift.com organization to deploy to
+    required: true
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      annotations:
+        component-display-name: ${DISPLAY_NAME}
+        component-name: ${REPO_NAME}
+      labels:
+        managed.openshift.io/gitHash: ${IMAGE_TAG}
+        managed.openshift.io/gitRepoName: ${REPO_NAME}
+        managed.openshift.io/osd: "true"
+      name: deployment-validation-operator
```

Comment:
Could you rename it `deployment-validation-operator-hypershift` to avoid a conflict with the existing `deployment-validation-operator` SelectorSyncSet